programs/coreutils/ls.cpp

```diff
@@ -19,56 +19,39 @@
 
 // A program that lists the contents of a directory.
 
-#include <cstdio>
 #include <libduck/Args.h>
 #include <libduck/Path.h>
-#include <unistd.h>
+#include <libduck/FormatStream.h>
 
-std::string dir_name = ".";
-bool no_color = false;
-bool colorize = false;
-bool show_all = false;
+#include <unistd.h>
 
-const char* REG_FORMAT = "\033[39m";
-const char* DIR_FORMAT = "\033[36m";
-const char* BLK_FORMAT = "\033[33m";
-const char* CHR_FORMAT = "\033[32m";
-const char* LNK_FORMAT = "\033[34m";
+int main(int argc, char** argv) {
+	std::string g_dir_name = ".";
+	bool g_no_color = false;
+	bool g_colorize = false;
+	bool g_show_all = false;
 
-int main(int argc, char **argv, char **env) {
 	Duck::Args args;
-	args.add_positional(dir_name, false, "DIR", "The directory to list.");
-	args.add_named(no_color, "n", "no-color", "Do not colorize the output.");
-	args.add_named(colorize, std::nullopt, "color", "Colorize the output.");
-	args.add_named(show_all, "a", "all", "Show entries starting with \".\".");
+	args.add_positional(g_dir_name, false, "DIR", "The directory to list.");
+	args.add_named(g_no_color, "n", "no-color", "Do not g_colorize the output.");
```

Comment:
```suggestion
	args.add_named(g_no_color, "n", "no-color", "Do not colorize the output.");
```

Accidental find+replace mishap :)